src/applications/vaos/reducers/newAppointment.js

```diff
@@ -578,18 +582,34 @@ export default function formReducer(state = initialState, action) {
         action.schema,
       );
 
+      let additionalInfoTitle;
+
+      // If community care, hide reasonForAppointment radios, update
+      // additional info title, remove expand condition and make optional
+      if (isCommunityCare) {
+        additionalInfoTitle = REASON_ADDITIONAL_INFO_TITLES.cc;
+        reasonSchema = unset('properties.reasonForAppointment', reasonSchema);
+        reasonSchema = unset('required', reasonSchema);
+        delete uiSchema.reasonAdditionalInfo['ui:options'].expandUnder;
```

Comment:
It's pretty unlikely, but I believe with this change, if you go through the CC flow, then go all the way back start back through the VA flow, this page won't work. You're making a shallow copy on line 566, but these are deleting data in sub-objects, which aren't copied. I could be wrong, definitely verify that before making any changes.

If that is true, you could change these deletes to `unset`, that might fix it. You could also have two different versions of the uiSchema in the page component and have the reducer switch between them depending on the flow.

Reply:
I tested going back and it seemed fine, but created separate schemas anyway